### ISSUE TITLE
testloader first argument

### ISSUE THREAD

**Issue by Spliomix** [MOhammedJAbi/Imsat#2]
Hi i tried your code with different datasets(cifar10) and some anomalies occur, i figured out that something in the testloader is wrong.

trainloader = torch.utils.data.DataLoader(trainset, batch_size=batch_size, shuffle=True, num_workers=2)
testloader = torch.utils.data.DataLoader(**trainset**, batch_size=batch_size, shuffle=False, num_workers=2)

On testloader at the first argument i think you should use the testset and not the trainset or does this have a deep meaning?
